ocaml/forkexecd/helper/vfork_helper.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) Citrix Systems Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published
+ * by the Free Software Foundation; version 2.1 only. with the special
+ * exception on linking described in file LICENSE.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ */
+
+#ifndef GUARD_hRiZStWEvmjkFidRuwMC9c
+#define GUARD_hRiZStWEvmjkFidRuwMC9c
```

Comment:
Looking at the rest of xapi, include guards are meaningful and human-readable and not randomly generated/hashed

Reply:
Unified to use `pragma once`.